roles/cloudera_deploy/tasks/init.yml

```diff
@@ -173,7 +173,13 @@
       name_prefix: "{{ name_prefix | default(default_name_prefix) }}"
       tags: "{{ tags | default(omit) }}"
       region: "{{ infra_region | default(default_infra_region) }}"
+      infra_deployment_engine: "{{ infra_deployment_engine | default(default_infra_deployment_engine) }}"
```

Comment:
It would be good to validate that terraform related values are presented in CD as expected. There are a few examples in init.yml for this.

Reply:
Good idea, I've added some validation now.